exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env ruby
 require 'minitest/autorun'
 require_relative 'all_your_base'
 
-# Common test data version: 1.0.0 39de23b
+# Common test data version: 1.1.0 c4d8d95
```

Comment:
Can you please list the ones I have to revert, or point me where I can find the canonical ones?